composer.json

```diff
@@ -12,7 +12,10 @@
     ],
     "require": {
         "php": ">=5.3.3",
```

Comment:
The PHP version needs to be increased for these changes. Could you make it on par with Symfony3 (`>=5.5.9`)?